src/stdlib/string.rs

```diff
@@ -0,0 +1,33 @@
+use gc_arena::MutationContext;
+use gc_sequence as sequence;
+
+use crate::{Callback, CallbackResult, Root, RuntimeError, String, Table, Value};
+
+pub fn load_string<'gc>(mc: MutationContext<'gc, '_>, _: Root<'gc>, env: Table<'gc>) {
+    let string = Table::new(mc);
+
+    string
+        .set(
+            mc,
+            String::new_static(b"len"),
+            Callback::new_sequence(mc, |args| {
+                Ok(sequence::from_fn_with(args, |mc, args| {
+                    match args
+                        .get(0)
+                        .cloned()
+                        .unwrap_or(Value::Nil)
+                        .to_string(mc)
+                    {
+                        Some(s) => Ok(CallbackResult::Return(vec![Value::Integer(s.len())])),
+                        _ => Err(RuntimeError(Value::String(String::new_static(
```

Comment:
Really small thing, I think just None => is easier to read here.

Reply:
I'll fess up: that's a copy-paste from math.rs. But I agree and will update.

Reply:
oh it's probably wrong there too :P